packages/frontend/src/features/metricsCatalog/hooks/useRunMetricExplorerQuery.ts

```diff
@@ -60,7 +63,11 @@ const postRunMetricExplorerQuery = async ({
             queryString ? `?${queryString}` : ''
         }`,
         method: 'POST',
-        body: undefined,
+        body: timeDimensionOverride
+            ? JSON.stringify({
+                  timeDimensionOverride,
+              })
+            : undefined,
```

Comment:
Oh, I see. Okay, maybe we should include a dimension id override with the grain as part of the string, and then just use it in the backend.

So far, what I see is that `TimeDimensionConfig` includes the name of the table, the field ID, and the grain.

This can be combined into a single field ID before making the request.

The override can be named `timeDimensionId` or `timeDimensionIdOverride` as a query parameter.

wdyt?

Reply:
so it would be like
'orders_order_id_month'?

Reply:
good shout, on it